angular/src/lib/sidebar/examples/sidebar-pageLevel.component.ts

```diff
@@ -4,18 +4,17 @@ import { Component } from '@angular/core';
   selector: 'example-sidebar-page-level',
```

Comment:
@pauljeter should these example components be camelCase or with kebab? Referring to the actual file name not the selector.

Reply:
good catch, file name should also be camel cased